internal/integration_test/fuzz/fuzz/fuzz_targets/validation.rs

```diff
@@ -18,6 +18,7 @@ fn run(data: &[u8]) -> Result<()> {
     // Generate the configuration with possibly invalid functions.
     let mut config = Config::arbitrary(&mut u)?;
     config.threads_enabled = true;
+    config.tail_call_enabled = true;
```

Comment:
this needs to be enabled for no_diff tests not only for the validation.

Reply:
should be done. If that is all (it's been a while 🙂) I reran the tests on arm64+amd64 and got no issues